examples/event_handler_graphql/src/assert_async_graphql_response_module.py

```diff
@@ -13,6 +6,7 @@
 from aws_lambda_powertools import Logger, Tracer
 from aws_lambda_powertools.event_handler import AppSyncResolver
 from aws_lambda_powertools.logging import correlation_paths
+from aws_lambda_powertools.shared.types import TypedDict
```

Comment:
was line highlighting affected in the docs?

Reply:
You are always saving our lives, Heitor! Yes, this refactors messed up the examples.

Done!